travis.sh

```diff
@@ -45,12 +45,18 @@ then
       rbenv install --verbose $ruby_version
     )
 
-    gem pristine --all
-    gem update --no-document --system
-    gem update --no-document
+    yes | gem pristine --all
+    yes | gem update --no-document --system --force
+    yes | gem update --no-document --force
+
+    if [ "$ruby_version" == "2.2.7" ] || [ "$ruby_version" == "2.1.10" ]; then
```

Comment:
I think you want to try this : https://github.com/SciRuby/daru-view/pull/131/commits/d6de2718707bce40f4b2d9a21888c65c3e27f422

Reply:
Yeah, that'd be a better way of doing it. Thanks.